src/pages/node/sessions.tsx

```
@@ -0,0 +1,377 @@
+  const tabLabel = 'incoming';
+  const channelsData = channels?.incoming;
+
+  const handleRefresh = () => {
+    if (!apiEndpoint) return;
+
+    dispatch(
+      actionsAsync.getSessionsThunk({
+        apiEndpoint,
+        apiToken: apiToken ? apiToken : '',
+        protocol: 'udp'
+      }),
```

ESLint

[error] Missing trailing comma.

(comma-dangle)